SubEthaEdit-Mac/Source/PlainTextDocument.h

```diff
@@ -89,6 +89,9 @@ extern NSString * const PlainTextDocumentDidSaveShouldReloadWebPreviewNotificati
     DocumentMode  *I_documentMode;
     FoldableTextStorage *I_textStorage;
     
+    // I_fileHash represents the MD5 hash of the document
+    NSData *I_fileHash;
```

Comment:
style: please put new ivars if possible in the implementation side, also switch to regular _ prefixes as we'd like to get away from the `I_`, `O_` etc. old school prefixes.